docs/api-docs/apps/guide/apps-03-developing.md

```diff
@@ -2,15 +2,119 @@
 
 <div class="otp" id="no-index">
 

```

Comment:
The section headings on this page should be [sentence case](https://intranet.bigcommerce.com/display/SST/Headings+and+Titles)